examples/histogram.jl

```diff
@@ -0,0 +1,119 @@
+using KernelAbstractions, Test
+using Atomix: @atomic, @atomicswap, @atomicreplace
```

Comment:
For consistency, I think it'd be better to either (1) use

```suggestion
using KernelAbstractions: @atomic, @atomicswap, @atomicreplace
```

*or* (2) simply document that people should use `using Atomix: @atomic, @atomicswap, @atomicreplace` and drop Atomix.jl from KernelAbstractions.jl dependency.

Reply:
Yeah, right now people have the choice to use either, which is nice. If @vchuravy wants to be more independent, I don't mind dropping the dependency, though.

Reply:
I am okay with having Atomix as dependency.